pom.xml

```diff
@@ -1451,7 +1606,7 @@ org.hibernate.SharedSessionContract#getTransaction()
                          <execution>
                              <id>verify-forbidden-test-apis</id>
                              <configuration>
-                                 <targetVersion>1.8</targetVersion>
+                                 <targetVersion>${maven.compiler.argument.target}</targetVersion>
```

Comment:
`maven.compiler.argument.testTarget` maybe? Since this particular execution is for tests only.
Also, I think you missed the same problem in the `verify-forbidden-apis` execution just above.

Reply:
Fixed, thanks.